e2e/e2e_docker_test.go

```diff
@@ -44,6 +44,9 @@ func runPipelineTests(t *testing.T, path string) {
 			if err != nil {
 				t.Fatalf("failed to create test runner: %s", err)
 			}
+			if r.Skip() {
+				t.Skip()
```

Comment:
So `r.Run` will still be invoked below. I am curious what will be the `skip` behavior then ?

Reply:
We can jump on a call to to avoid back and forth on it.

Reply:
So I looked at the testing pkg https://golang.org/src/testing/testing.go?s=28687:28729#L810 and `t.Skip()` exits the `GoRoutine` that the test is running in, so that answers my question.